requirements.txt

```diff
@@ -3,3 +3,5 @@ Jinja2<3.0
 urlobject<3.0
 lxml<4.0
 path.py<9.0
+requests<3.0
+pyyaml<4.0
```

Comment:
edx-platform pins PyYAML to 3.10, and requests to 2.9.1. They should both be compatible with this version specifier.